google-cloud-storage/src/main/java/com/google/cloud/storage/BucketInfo.java

```diff
@@ -1731,7 +1731,7 @@ public Long getMetageneration() {
 
   /**
    * Returns the bucket's location. Data for blobs in the bucket resides in physical storage within
-   * this region.
+   * this region or regions.
    *
    * @see <a href="https://cloud.google.com/storage/docs/bucket-locations">Bucket Locations</a>
```

Comment:
Does dual-region have a link that we can direct folks to our is it just the bucket-locations doc? 

cc: @danielbankhead

Reply:
That will be here: https://cloud.google.com/storage/docs/locations

(the updates are currently in staging, pending for release)

Reply:
it appears the link `https://cloud.google.com/storage/docs/bucket-locations` redirects to `https://cloud.google.com/storage/docs/locations` I can update this link.